_posts/en/newsletters/2021-12-22-newsletter.md

```diff
@@ -0,0 +1,665 @@
+that contain all the information necessary to allow a wallet or other
+program to track payments made to or spent from a particular script or
+set of related scripts (i.e. an address or a set of related addresses
+such as in an HD wallet).  Descriptors combine well with [miniscript][topic miniscript] in
+allowing a wallet to handle tracking and signing for a larger variety of
+scripts.  They also combine well with [PSBTs][topic psbt] in allowing
+the wallet to determine which keys it controls in a multisig script.  By
+the end of the year, Bitcoin Core had made descriptor-based wallets its
+[default][descriptor default] for newly-created wallets.
+
+A common way of opening LN channels that had never before been official
+supported began to be [specified][0conf channels] in July.  Zero-conf
+channel opens, also called *turbo channels*, are new single-funded
+channels where the funder gives some or all of their initial funds to
+the acceptor. Those funds are not secure until the channel open
+transaction receives a sufficient number of confirmations, so there’s no
+risk to the acceptor spending some of those funds back through the
+funder using the standard LN protocol.  For example, Alice has several
+BTC in an account at Bob’s custodial exchange. Alice asks Bob to open a
+new channel paying her 1.0 BTC. Because Bob trusts himself not to
+double-spend the channel he just opened, he can allow Alice to send 0.1
+BTC through his node to third-party Carol even before the channel open
+transaction has received a single confirmation.  Specifying the behavior
+should help improve interoperability between LN nodes and the merchants
+who offer this service.
+
+{:.center}
+![Zero-conf channel illustration](/img/posts/2021-07-zeroconf-channels.png)
+
+Two related proposals for new signature hash (sighash) types were
+[combined][sighash combo] into [BIP118][].  `SIGHASH_NOINPUT`, proposed
+in 2017 and partly based on previous proposals going back a decade, was
+superseded by [`SIGHASH_ANYPREVOUT`][topic sighash_anyprevout] first
+[proposed][anyprevout proposed] in 2019.  The new sighash types will
+allow offchain protocols such as LN and [vaults][topic vaults] to reduce
+the amount of intermediate states they need to retain, greatly reducing
+storage requirements and complexity.  For multiparty protocols, the
+benefits may be even more significant by eliminating the number of
+different states that need to be generated in the first place.
+
+<!--
+TODO:EXTRA LN migration to real databases:
+  /en/newsletters/2021/07/14/#lnd-5447
+  /en/newsletters/2021/07/21/#eclair-1871
+  /en/newsletters/2021/10/06/#lnd-5366
+-->
+
+## August
+
+Fidelity bonds is an idea [proposed][wiki contract] at least as early as
+2010 for locking up bitcoins for a period of time in order to create a
+cost for misbehavior.  Because the bitcoins can't be used again until
+their time lock expires, anyone banned or otherwise penalized during the
```

Comment:
Perhaps it's worth distinguishing this as an already-implementable idea for applications, and not a proposal for a protocol change (which much of the other things called "proposal" in this newsletter are). I'm imagining a (not-well-informed) reader that gets confused thinking there's a proposal for banning users temporarily.
```suggestion
Since at least as early as 2010, fidelity bonds have been [discussed][wiki contract]
as a way for Bitcoin applications to penalize misbehavior by locking up
bitcoins for a period of time.  Because the bitcoins can't be used again until
their time lock expires, anyone banned or otherwise penalized during the
```